wolfcrypt/src/kdf.c

```diff
@@ -435,8 +435,29 @@ int wc_PRF_TLS(byte* digest, word32 digLen, const byte* secret, word32 secLen,
     {
         int    ret = 0;
         int    idx = 0;
-        byte   data[MAX_TLS13_HKDF_LABEL_SZ];
+        byte   dataStatic[MAX_TLS13_HKDF_LABEL_SZ];
+        byte   *data;
 
+        /* okmLen 16 */
+        idx = 2;
+        /* protol label len */
+        idx += 1;
+        idx += protocolLen;
+        idx += labelLen;
+        /* info len */
+        idx += 1;
+        idx += infoLen;
+
+        /* it may happen with long nonce value */
+        if (idx > MAX_TLS13_HKDF_LABEL_SZ)
+            data = XMALLOC(idx, NULL, DYNAMIC_TYPE_TMP_BUFFER);
```

Comment:
For customers using static memory with WOLFSSL_NO_MALLOC they will want the heap hint here. You might need to refactor a _ex version that support heap hint `void* heap`.

Reply:
In the end, I had to use a new function inside `tls13.c`